.env

```diff
@@ -16,3 +16,12 @@ BRUPOP_CONTAINER_IMAGE=public.ecr.aws/bottlerocket/bottlerocket-update-operator:
 # brupop will exclude the node from load balancer and 
 # wait for EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC seconds before draining node.
 EXCLUDE_FROM_LB_WAIT_TIME_IN_SEC=0
+
+# Concurrent update nodes setting.
```

Comment:
I didn't see any default value for this env variable, should there be one?

Reply:
The default value for the variable is 1, its in line 27.